src/core/wall.ts

```diff
@@ -37,10 +49,24 @@ export interface StoredWallState  {
 
 // List of maps, stores the minimal amount of data needed so that
 // this could be moved into a JSON file to backup maps
-export const MapCatalog: StoredWallState[][] = [[
-  {pos: {x: 380, y: 260}, width: 200, height: 200, sprite: WallSprite.BUNKER_2x2_1},
-  {pos: {x: 170, y: 145}, width: 140, height: 70, sprite: WallSprite.BUNKER_2x1_1},
-  {pos: {x: 205, y: 470}, width: 70, height: 140, sprite: WallSprite.BUNKER_1x2_1},
-  {pos: {x: 650, y: 505}, width: 140, height: 70, sprite: WallSprite.BUNKER_2x1_2},
-  {pos: {x: 685, y: 110}, width: 70, height: 140, sprite: WallSprite.BUNKER_1x2_1},
-]];
+export const MapCatalog = [{
+  walls: [
+    {pos: {x: 380, y: 260}, width: 200, height: 200, sprite: WallSprite.BUNKER_2x2_1},
+    {pos: {x: 170, y: 145}, width: 140, height: 70, sprite: WallSprite.BUNKER_2x1_1},
+    {pos: {x: 205, y: 470}, width: 70, height: 140, sprite: WallSprite.BUNKER_1x2_1},
+    {pos: {x: 650, y: 505}, width: 140, height: 70, sprite: WallSprite.BUNKER_2x1_2},
+    {pos: {x: 685, y: 110}, width: 70, height: 140, sprite: WallSprite.BUNKER_1x2_1},
+    {pos: {x: 0, y: 720}, width: 960, height: 50, sprite: WallSprite.BUNKER_2x1_2},
```

Comment:
Could you move this into `Game.init` and make it use the `width`/`height`, since this'll be applicable to any map.